I still get webhook tests failing on macos 10.15, e.g. `TestWebhook.test_send_attachments`:

```
...
>               return self._sock.recv_into(b)
E               ConnectionResetError: [Errno 54] Connection reset by peer

/nix/store/9h7xr11my83hh41m5p1cidnjywa67i2p-python3-3.8.12/lib/python3.8/socket.py:669: ConnectionResetError
----------------------------- Captured stderr call -----------------------------
127.0.0.1 - - [06/Nov/2021 13:36:35] "POST / HTTP/1.1" 200 -
----------------------------------------
Exception happened during processing of request from ('127.0.0.1', 51525)
Traceback (most recent call last):
  File "/nix/store/9h7xr11my83hh41m5p1cidnjywa67i2p-python3-3.8.12/lib/python3.8/socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/nix/store/9h7xr11my83hh41m5p1cidnjywa67i2p-python3-3.8.12/lib/python3.8/socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
  File "/nix/store/9h7xr11my83hh41m5p1cidnjywa67i2p-python3-3.8.12/lib/python3.8/socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/nix/store/9h7xr11my83hh41m5p1cidnjywa67i2p-python3-3.8.12/lib/python3.8/http/server.py", line 647, in __init__
    super().__init__(*args, **kwargs)
  File "/nix/store/9h7xr11my83hh41m5p1cidnjywa67i2p-python3-3.8.12/lib/python3.8/socketserver.py", line 747, in __init__
    self.handle()
  File "/nix/store/9h7xr11my83hh41m5p1cidnjywa67i2p-python3-3.8.12/lib/python3.8/http/server.py", line 427, in handle
    self.handle_one_request()
  File "/nix/store/9h7xr11my83hh41m5p1cidnjywa67i2p-python3-3.8.12/lib/python3.8/http/server.py", line 416, in handle_one_request
    self.wfile.flush() #actually send the response if not already done.
ValueError: I/O operation on closed file.
----------------------------------------
------------------------------ Captured log call -------------------------------
2021-11-06 13:36:35 DEBUG Sending a request - url: http://localhost:8888, body: {"text": "hello!", "attachments": [{"color": "#f2c744", "blocks": [{"type": "section", "text": {"type": "mrkdwn", "text": "This is a mrkdwn section block :ghost: *this is bold*, and ~this is crossed out~, and <https://google.com|this is a link>"}}, {"type": "divider"}, {"type": "section", "text": {"type": "mrkdwn", "text": "Pick a date for the deadline."}, "accessory": {"type": "datepicker", "initial_date": "1990-04-28", "placeholder": {"type": "plain_text", "text": "Select a date"}}}]}], "response_type": "ephemeral"}, headers: {'Content-Type': 'application/json;charset=utf-8', 'User-Agent': 'Python/3.8.12 slackclient/2.9.3 Darwin/19.6.0'}
2021-11-06 13:36:35 ERROR Failed to send a request to Slack API server: [Errno 54] Connection reset by peer
```

--- a/pkgs/development/python-modules/slackclient/default.nix
+++ b/pkgs/development/python-modules/slackclient/default.nix
@@ -1,4 +1,5 @@
 { lib
+, stdenv
 , aiohttp
 , buildPythonPackage
 , codecov
@@ -51,7 +52,18 @@ buildPythonPackage rec {
 
   # Exclude tests that requires network features
   pytestFlagsArray = [ "--ignore=integration_tests" ];
-  disabledTests = [ "test_start_raises_an_error_if_rtm_ws_url_is_not_returned" ];
+
+  disabledTests = [
+    "test_start_raises_an_error_if_rtm_ws_url_is_not_returned"
+  ] ++ lib.optionals stdenv.isDarwin [
+    # these fail with `ConnectionResetError: [Errno 54] Connection reset by peer`
+    "test_issue_690_oauth_access"
+    "test_issue_690_oauth_v2_access"
+    "test_send"
+    "test_send_attachments"
+    "test_send_blocks"
+    "test_send_dict"
+  ];
 
   pythonImportsCheck = [ "slack" ];
 

